manifests/init.pp

```diff
@@ -23,9 +23,12 @@
   $cron_d_file        = '/etc/cron.d/rancid',
   $cloginrc_content   = 'USE_DEFAULTS',
   $show_cloginrc_diff = true,
+  $vcs                = 'USE_DEFAULTS',
```

Comment:
The default would ideally be the actual default like `cvs` here, though I see you are following the existing pattern so ok to leave this alone.